src/rgtk.rs

```diff
@@ -60,8 +60,9 @@ extern crate "cairo-sys" as cairo_ffi;
 
 extern crate glib;
 
+pub use glib::Type;
+pub use glib::{GValue, GValuePublic};
```

Comment:
I agree that expiring the crate would be better, but I didn't know how. If
I do `pub mod glib` I get a name clash.
On Mar 6, 2015 4:30 AM, "Gleb Kozyrev" <notifications@github.com> wrote:

> In src/rgtk.rs
> <https://github.com/jeremyletang/rgtk/pull/226#discussion_r25933804>:
>
> > @@ -60,8 +60,9 @@ extern crate "cairo-sys" as cairo_ffi;
> >
> >  extern crate glib;
> >
> > +pub use glib::Type;
> > +pub use glib::{GValue, GValuePublic};
>
> Hmm, if you're doing this for convenience of the example, maybe just
> reexport glib instead of cherry-picking its types? I seemed to see an
> attribute that wouldn't let its documentation spill into this crate's when
> doing that...
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/jeremyletang/rgtk/pull/226/files#r25933804>.
>